WordPress/gutenberg
# Support reusable blocks in widgets editor

## Description
Related to https://github.com/WordPress/gutenberg/issues/22875

This adds support for inserting reusable blocks into widgets editor. Support for creating new reusable blocks is not included in this PR.

## How has this been tested?
1. Make sure you have some reusable blocks, if you don't then create a few in the post editor
1. Go to the new widgets editor
1. Open inserter
1. Confirm it has "Reusable blocks" tab
1. Confirm the tab contains all your reusable blocks and that you're able to insert them into the editor

## Types of changes
New feature (non-breaking change which adds functionality)

## Checklist:
